### PR TITLE
Documentation for TimeZone feature, pointing to release/6.6.0. Addressed all code review comments provided in https://github.com/stleary-cisco/ftd-rest-api-cookbook/pull/5

### PR DESCRIPTION
**Pull request by Arunima19**
…sed code review comments

### CODE REVIEW

**Comment by Arunima19** [comment]
@stleary-cisco Addressed all review comments from https://github.com/stleary-cisco/ftd-rest-api-cookbook/pull/5. Facing some issues in https://github.com/stleary-cisco/ftd-rest-api-cookbook/pull/5. Closing the old PR https://github.com/stleary-cisco/ftd-rest-api-cookbook/pull/5.

File: examples/ftd_api_scripts/timezone.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+from ftd_api_resources.timezone_utils import get_all_time_zones, post_time_zone, update_time_zone, get_time_zone, \
+    delete_time_zone
+from ftd_api_resources.access_token import get_access_token
+
+
+def time_zone_actions(host, port, user, passwd):
+    """
+    End to end example of code that updates a time-zone object.
+    Requires Python v3.0 or greater and the reqeusts library.
+    You must update the values for host, port, user, and password to connect to your device.
```
Comment (stleary-cisco) [inline examples/ftd_api_scripts/timezone.py]: Please include params in docstring

File: examples/ftd_api_scripts/timezone.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    if not returnCode:
+        print('Unable to delete time zone object')
+        return
+    return True
+
+
+if __name__ == '__main__':
+
+    if len(sys.argv) != 5:
+        print("Usage: python cookbook_scripts/timezone.py host port user passwd")
```
Comment (stleary-cisco) [inline examples/ftd_api_scripts/timezone.py]: cookbook_scripts -> ftd_api_scripts

File: examples/ftd_api_scripts/timezoneobject_timezonesetting_association.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+from ftd_api_resources.timezone_utils import post_time_zone
+from ftd_api_resources.access_token import get_access_token
+
+
+def timezoneobject_timezonesetting_associated_actions(host, port, user, passwd):
+    """
+    End to end example of code that updates the timezone-setting.
+    Requires Python v3.0 or greater and the reqeusts library.
+    You must update the values for host, port, user, and password to connect to your device.
+    """
```
Comment (stleary-cisco) [inline examples/ftd_api_scripts/timezoneobject_timezonesetting_association.py:31]: Please doc params

File: examples/ftd_api_scripts/timezoneobject_timezonesetting_association.py [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+        print('Unable to get timezone setting')
+        return False
+    print('TimeZone Setting name and Id are {} {}'.format(tzs['name'], tzs['id']))
+    return True
+
+
+if __name__ == '__main__':
+
+    if len(sys.argv) != 5:
+        print("Usage: python cookbook_scripts/timezoneobject_timezonesetting_association.py host port user passwd")
```
Comment (stleary-cisco) [inline examples/ftd_api_scripts/timezoneobject_timezonesetting_association.py]: cookbook_scripts -> ftd_api_scripts

File: examples/ftd_api_scripts/timezone.py (hunk 1 above)
Comment (Arunima19) [reply examples/ftd_api_scripts/timezone.py]: Done

File: examples/ftd_api_scripts/timezone.py (hunk 2 above)
Comment (Arunima19) [reply examples/ftd_api_scripts/timezone.py]: Done

File: examples/ftd_api_scripts/timezoneobject_timezonesetting_association.py (hunk 3 above)
Comment (Arunima19) [reply examples/ftd_api_scripts/timezoneobject_timezonesetting_association.py:31]: Done

File: examples/ftd_api_scripts/timezoneobject_timezonesetting_association.py (hunk 4 above)
Comment (Arunima19) [reply examples/ftd_api_scripts/timezoneobject_timezonesetting_association.py]: Done

**Review by jaredtsmith — APPROVED** [review]

**Review by stleary-cisco — APPROVED** [review]

**Pull request merged**